mpd/asyncio.py

```diff
@@ -273,82 +272,33 @@ async def __read_output_line(self):
             return None
         return line
 
-
-#    async def _parse_objects_direct(self, lines, delimiters=[], lookup_delimiter=False):
-#        obj = {}
-#        while True:
-#            line = await lines.get()
-#            if isinstance(line, BaseException):
-#                raise line
-#            if line is None:
-#                break
-#            key, value = self._parse_pair(line, separator=": ")
-#            key = key.lower()
-#            if lookup_delimiter and not delimiters:
-#                delimiters = [key]
-#            if obj:
-#                if key in delimiters:
-#                    yield obj
-#                    obj = {}
-#                elif key in obj:
-#                    if not isinstance(obj[key], list):
-#                        obj[key] = [obj[key], value]
-#                    else:
-#                        obj[key].append(value)
-#                    continue
-#            obj[key] = value
-#        if obj:
-#            yield obj
-
-    def _parse_objects_direct(self, lines, delimiters=None, lookup_delimiter=False):
-        # This is a workaround implementing the above comment on Python 3.5. It
-        # is recommended that the commented-out code be used for reasoning, and
-        # that changes are applied there and only copied over to this
-        # implementation.
-
-        outerself = self
-        class WrappedLoop:
```

Comment:
@chrysn @xcant I just uncomment the old code now that 3.5 is dropped as supported version. @xcant please check if https://github.com/Mic92/python-mpd2/pull/117 is not re-introduced.

Reply:
I just cloned the repository and it works fine with the code where I detected the error, so #117 is not re-introduced.